espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/http/RouteDefinition.java

```diff
@@ -0,0 +1,78 @@
+package io.appium.espressoserver.lib.http;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import fi.iki.elonen.NanoHTTPD.Method;
+import io.appium.espressoserver.lib.handlers.RequestHandler;
+import io.appium.espressoserver.lib.model.AppiumParams;
+
+class RouteDefinition {
+
+    private final String testRegex;
+    private final String routeUri;
+    private final Method method;
```

Comment:
Method is part of the `fi.iki.elonen.NanoHTTPD.Method` package

It refers to GET, POST, PUT, DELETE, etc...